### ISSUE TITLE
Error clicking on an object in Avocode Inpector mode

### ISSUE THREAD

**Issue by travisedwardbrown** [avocode/avocode#2207]
---
* **Version**: 3.6.0-web.2
* **Web/Desktop**: Web
* **Platform**: macosx 10.14
* **Browser**: safari 12.0
* **Avocode UID**: 440265
---
```
undefined is not an object (evaluating 'e.set')
https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:156:1067314
e@https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:15535
e@https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:15624
e@https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:15624
updateIn@https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:15687
https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:156:1067287
__iterate@https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:26040
_handleSelectionStoreChange@https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:156:1067236
emit@https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:32:132567
value@https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:105:40460
_handleItemSelectAction@https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:156:1516041
_invokeCallback@https://cdn.app.avocode.c
```

**Comment by crashmajstr** [comment]
Hi @travisedwardbrown  , we're currently working on a fix, I am sorry for the inconvenience, we'll let you know once it's fixed.
Closing with respect to #2202
Best,
Josef

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ll release the fix in an upcoming hour or so. It’ll be version 3.6.1.
For now, you can just download and use our older stable build 3.5.6 

Download links
Mac: https://we.tl/t-bS4jyZufpT
Win: https://we.tl/t-12nDKmpafd
Linux: https://we.tl/t-krrbw7ZRvB

I’ll let you know once the fix is out. Sorry for the inconvenience.

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ve just released the new version of Avocode 3.6.1 containing the fix for this issue. 
Please, update your desktop app, if you use the web app the new version is already there.
Hope everything will be ok from now on.
I am very sorry for the inconvenience. 
On behalf of Avocode,
Josef Kettner
